meta-mbl-distro/scripts/lib/mbl/update/appsimage.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""PayloadImage subclass for apps images."""
+
+import arpy
```

Comment:
I noticed we are just going to install this and `libconf` in our docker container. Even so I would prefer we provided a `requirements.txt` for users who don't use the mbl-tools scripts.

Reply:
Okay.

Reply:
Added requirements.txt in https://github.com/ARMmbed/meta-mbl/pull/763/commits/efb13e2bdcf34f162330be0acadc579fe1b30b37